scalafix-reflect/src/main/scala/scalafix/internal/reflect/GitHubUrlRule.scala

```diff
@@ -32,17 +36,21 @@ object GitHubUrlRule {
   private def guessGitHubURL(
       org: String,
       repo: String,
-      filename: String,
+      rule: String,
```

Comment:
maybe `className` (here and above) to highlight that `.` is used as separator and that extension is not provided? 
```suggestion 
      className: String,
```